condensedinlinepanel/wagtail_hooks.py

```diff
@@ -1,4 +1,7 @@
-from django.contrib.staticfiles.templatetags.staticfiles import static
+try:
```

Comment:
Please could you replace this with an explicit version check?

```python
import django

if django.VERSION >= (3, 0):
    from django.contrib.staticfiles.templatetags.staticfiles import static
else:
    from django.templatetags.static import static
```

This would make it obvious to our future-selves what this check is for.